invenio_rdm_records/services/communities/service.py

```diff
@@ -56,7 +168,7 @@ def _remove(self, identity, community_id, record):
         record.parent.communities.remove(community_id)
 
     @unit_of_work()
-    def remove(self, identity, record_id, data, revision_id=None, uow=None):
+    def remove(self, identity, record_id, data, uow):
```

Comment:
question: by making the `uow` mandatory, could  this break other code? E.g. https://github.dev/ntarocco/invenio-rdm-records/blob/230d77ed54eaf282916a1d570c5c93dc06afa6f4/invenio_rdm_records/services/community_records/service.py#L82